src/openrct2/localisation/FormatCodes.cpp

```diff
@@ -65,45 +63,26 @@ static const EnumMap<FormatToken> FormatTokenMap = {
 };
 // clang-format on
 
-std::string_view GetFormatTokenStringWithBraces(FormatToken token)
-{
-    // Ensure cache is thread safe
-    static std::mutex mutex;
-    std::lock_guard<std::mutex> guard(mutex);
-
-    static std::vector<std::string> cache;
-    auto index = static_cast<size_t>(token);
-    if (cache.size() <= index)
-    {
-        cache.resize(index + 1);
-    }
-    if (cache[index].empty())
-    {
-        cache[index] = "{" + std::string(FormatTokenToString(token)) + "}";
-    }
-    return cache[index];
-}
-
 FormatToken FormatTokenFromString(std::string_view token)
 {
     auto result = FormatTokenMap.find(token);
     return result != std::end(FormatTokenMap) ? result->second : FormatToken::Unknown;
 }
 
-std::string_view FormatTokenToString(FormatToken token, bool withBraces)
+std::string FormatTokenToString(FormatToken token)
 {
-    if (withBraces)
-    {
-        return GetFormatTokenStringWithBraces(token);
-    }
-
     auto it = FormatTokenMap.find(token);
     if (it != FormatTokenMap.end())
-        return it->first;
+        return std::string(it->first);
 
     return {};
 }
 
+std::string FormatTokenToStringWithBraces(FormatToken token)
+{
+    return std::string("{") + FormatTokenToString(token) + std::string("}");
```

Comment:
I realise now that this was a workaround for the MinGW bug. We can probably just do this again instead:
```suggestion
    return "{" + FormatTokenToString(token) + "}";
```

I'll rebase after #21468 has been merged.